openedx/core/djangoapps/user_api/management/tests/test_email_opt_in_list.py

```diff
@@ -411,6 +415,8 @@ def _assert_output(self, output, *args, **kwargs):
         # Check data rows
         for user, course_id, opt_in_pref in args:
             self.assertIn({
```

Comment:
Originally, there are multiple dicts/rows being passed in output in certain tests, thus requiring this `assertIn`. `assertEqual` fails here. 

If you would like to dig deeper/refactor the tests, I suggest we create a separate ticket.